.github/workflows/pr.yml

```diff
@@ -38,8 +38,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Generate layout.json and update manifest.json
         run: |
-          ./scripts/dev-env/run.sh ./scripts/setup.sh
-          ./scripts/dev-env/run.sh ./scripts/build.sh
+          ./scripts/dev-env/run.sh npm install
```

Comment:
This is wrong. it needs the `--no-optional` and `--legacy-peer-deps`. See comment above.